roles/devscripts/tasks/main.yml

```diff
@@ -41,22 +41,45 @@
           (cifmw_devscripts_ocp_online | bool)
         )
       )
-  register: _devscripts_deploy
-  community.general.make:
-    chdir: "{{ cifmw_devscripts_repo_dir }}"
-    target: "all"
-  async: "{{ cifmw_devscripts_installer_timeout }}"
-  poll: 0
+  vars:
+    _log_src: >-
```

Comment:
Setting this var here makes the reviewer/future dev think it's used more than once but it's only used in line 72. Not a blocker.

Reply:
right, yeah. I used it to debug the output to be sure, and it stayed. Not a big deal imho indeed. Also, I wasn't sure about using `find` to list things, sooo... well. you know how it is ;).